mesonbuild/mparser.py

```diff
@@ -507,13 +546,14 @@ def e7(self):
         return left
 
     def e8(self):
+        block_start = self.current
```

Comment:
__[flake8]__
* [F841] local variable 'block_start' is assigned to but never used

<!-- Comment by [SideCI](https://sideci.com) -->